src/common/services/resPersonApi.js

```diff
@@ -0,0 +1,102 @@
+const request = require('request');
+const logger = require('../utils/logger')(__filename);
+const endpoints = require('../config/endpoints');
+
+module.exports = {
+
+    create(userId, resPerson) {
+        logger.debug(JSON.stringify(resPerson));
```

Comment:
`Question`: Logging this can reveal `JSON.stringify(resPerson)` details e.g. `email address` `phone number` `firstName`, etc, etc.
* Is this an ok risk or not?
* I don't mind either way, as long as the risk has been accounted.

Reply:
This log is not be needed, removed.

Reply:
This is good.